src/main/java/com/project/service/UserServiceImpl.java

```diff
@@ -46,24 +45,24 @@ public User addUser(User user) throws UserException {
         user.setId(null);//do not allow choosing id
 
         user.setPassword(bcryptEncoder.encode(user.getPassword()));
-        if(userRepository.findByEmail(user.getEmail()).isPresent())
+        if (userRepository.findByEmail(user.getEmail()).isPresent())
             throw new UserEmailExistsException(user.getEmail());
 
         List<String> roles = new ArrayList<>();
-        roles.add("User");
+        roles.add("USER");
         user.setRoles(roles); //probably more logical as we don't have admin registration
```

Comment:
Don't think this comment should be here.

Reply:
removed unnecessary comments